presqt/fixity.py

```diff
@@ -0,0 +1,65 @@
+import hashlib
+
+def fixity_checker(binary_file, hashes):
+    """
+    Take a file in binary format and a dictionary of hashes and run a fixity check against the first
+    one found that's supported by hashlib.
+
+    Parameters
+    ----------
+    binary_file : byte object
+        File as a byte object.
+    hashes : dictionary
+        Dictionary of known hashes the file had in the target.
```

Comment:
... file had in the *source*?

I think you had mentioned wanting to use source/destination language instead of target.